server/data/dr.strange.json.json

```diff
@@ -9,14 +9,14 @@
   "name": "Doctor Strange",
   "photoUrl": "https://i.pinimg.com/564x/1a/33/29/1a3329a57b88c0326820eca80df2a4cb.jpg",
   "schoolId": 0,
-  "schoolName": "",
-  "schoolVerified": false,
-  "shareStatistics": false,
+  "schoolName": "test school name",
+  "schoolVerified": true,
+  "shareStatistics": true,
   "strandGrades": {
     "9,12,11": 0,
     "9,12,12": 0.2
   },
-  "superPower": "",
+  "superPower": "test super power",
```

Comment:
This can be an edit box, so the user can change the user power description text as he/she wishes.